app/models/authority.rb

```diff
@@ -220,7 +220,8 @@ def morph_url
 
   def morph_feed_url_for_date_range(start_date, end_date)
     query = CGI.escape("select * from `data` where `date_scraped` >= '#{start_date}' and `date_scraped` <= '#{end_date}'")
-    "https://api.morph.io/#{morph_name}/data.json?query=#{query}&key=#{ENV['MORPH_API_KEY']}"
+    key = CGI.escape ENV['MORPH_API_KEY'] || ""
```

Comment:
Defaulting this to an empty string doesn't feel like the right thing to do here, especially not to just [fix a test](https://github.com/openaustralia/planningalerts/pull/1139/commits/154f3f0eeb9fb9ede76dfcb45839e2feb6f0ea1a). Since morph.io requires an API key it seems reasonable to throw an exception if it's not set.

To get the tests passing it seems like we don't need a valid API key (there wasn't one before) but we do need to set one. So we can just add an obviously fake one to `.env.test`.